init-governance.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+set -euo pipefail
+
 solana -v --url http://localhost:8899 --keypair artifacts/creator.keypair airdrop 100
```

Comment:
Remove `--url http://localhost:8899`. Let `solana` and `spl-token` use URL from config.

Reply:
Fixed